app/src/main/java/com/doctor/yumyum/presentation/ui/write/WriteFragment4.kt

```diff
@@ -61,10 +61,10 @@ class WriteFragment4 : BaseFragment<FragmentWriteFourthBinding>(R.layout.fragmen
             detailTasteAdapter.updateSelectedList(it)
             if(it.isNullOrEmpty()){
                 binding.writeFourthBtnNext.isEnabled = false
-                binding.writeFourthBtnNext.background = resources.getDrawable(R.drawable.bg_btn_sub)
+                binding.writeFourthBtnNext.background = activity?.getDrawable(R.drawable.bg_btn_sub)
             }else{
                 binding.writeFourthBtnNext.isEnabled = true
-                binding.writeFourthBtnNext.background = resources.getDrawable(R.drawable.bg_btn_main)
+                binding.writeFourthBtnNext.background = activity?.getDrawable(R.drawable.bg_btn_main)
```

Comment:
non-nullable하게 처리해주는 게 좋을 것 같아요! 이런 식으로 하면 되려나요
```suggestion
                binding.writeFourthBtnNext.background = requireContext().getDrawable(R.drawable.bg_btn_main)
```

Reply:
맛선택 뒤로갔다 왔을때 튕기는거 때문에 수정한 부분인데 `requireContext()` 로 작성했을 때  `Fragment not attached to a context.` 에러가 떠서 위처럼 수정하였습니다

Reply:
헉 그건 그거대로 또 이상한데요... 우선 알겠습니다! 나중에 다시 봐용